Gopkg.lock

```diff
@@ -406,9 +406,12 @@
   version = "v0.3.3"
 
 [[projects]]
-  digest = "1:bbe569477853f0bd2e49e8f214ab37981293683dff4a9815e95a1465c3918ee6"
+  digest = "1:aef2bfdbd53d02eb4f346775b9b78dbc15011092497a049bb5b433dfe3634847"
   name = "k8s.io/kubernetes"
-  packages = ["pkg/util/mount"]
+  packages = [
+    "pkg/util/mount",
+    "pkg/volume/csi/csiv0",
```

Comment:
The csi spec should already be vendored in.  Do we really need to pull in the version from K8s?

Reply:
Ah I forgot to change the path after moving the tool location. We don't need to pull it from K8s.